src/Module/Admin/DBSync.php

```diff
@@ -102,13 +102,13 @@ protected function content(array $request = []): string
 
 				if (!count($failed)) {
 					$o = Renderer::replaceMacros(Renderer::getMarkupTemplate('admin/dbsync/structure_check.tpl'), [
-						'$base' => DI::baseUrl()->get(true),
+						'$base' => DI::baseUrl(),
```

Comment:
Same remark as for the `'$baseurl'` template variable, except it should be renamed in `admin/dbsync/structure_check.tpl`
```suggestion
```

Reply:
funny enough .. it isn't used in this template, it's already `$baseurl` everywhere, so we can remove this setting :)